server/pulp/plugins/util/publish_step.py

```diff
@@ -243,8 +241,9 @@ def process(self):
                         # Clean out the progress_details for the individual item
                         self.progress_details = ""
                     if self.exceptions:
-                        raise PulpCodedTaskFailedException(error_code=error_codes.PLP0032,
-                                                           task_id=self.status_conduit.task_id)
+                        msg = "Task %(task_id)s encountered one or more failures " \
```

Comment:
It's very important that this be reverted. We need the exception type raised to be of type `PulpCodedTaskFailedException` because since it happens in a Task raising this type causes the task status to update, and attach a structured error report to the `TaskStatus`.